src/__tests__/indexing.ts

```diff
@@ -290,17 +291,20 @@ describe('indexing', () => {
             { anchor: false }
           )
 
-          await TestUtils.delay(10 * 1000)
+          await waitForMidsToBeIndexed(ceramic2, [doc1, doc2])
 
           const did1Results = await ceramic2.index
             .query({ model: TEST_MODEL, last: 100, account: did1.id })
             .then(resultObj => extractDocuments(ceramic2, resultObj))
           expect(did1Results.length).toBeGreaterThanOrEqual(1)
 
-          const lastDid1Doc = did1Results.at(-1) as ModelInstanceDocument
-          expect(lastDid1Doc.id.toString()).toEqual(doc1.id.toString())
-          expect(lastDid1Doc.content).toEqual(doc1.content)
-          expect(StreamUtils.serializeState(lastDid1Doc.state)).toEqual(
+          // We cannot expect that the most recent MID will be the MID created by this test
+          // This is because we may receive and process pubsub messages for docs created by ceramicClient
```

Comment:
what do you mean by "docs created by ceramicClient" here?

Reply:
The test that is failing is `Can filter by DID` using `ceramic` (local). The test that runs before this is `Can create and query on same node` using the `ceramicClient` (remote node). Since pubsub messages can be received in a random order, sometimes the `ceramic` instance receives and indexes docs created by the `ceramicClient` from other tests. 

Actually this probably happens with `ceramic` and `ceramicClient` swapped. I'll change the comment to reflect this

Reply:
think you forgot to update this comment?  There's another comment above, on line 206, that looks better now